crates/utils/src/lib.rs

```diff
@@ -50,6 +50,9 @@ pub const CACHE_DURATION_LARGEST_COMMUNITY: Duration = DAY;
 
 pub const MAX_COMMENT_DEPTH_LIMIT: usize = 50;
 
+/// Doing DB transactions of bigger batches than this tend to cause seq scans.
+pub const DB_BATCH_SIZE: usize = 1000;
```

Comment:
Should be i64 then you dont need try_into

Reply:
Fixed